index.js

```diff
@@ -0,0 +1,72 @@
+document.addEventListener("DOMContentLoaded", async () => {
+    let img = document.querySelector("img")
+    let select = document.querySelector("select")
+    let movieInfoDiv = document.querySelector("#movieInfo")
+    let reviews = document.querySelector("#submittedReviews")
+    let form = document.querySelector("form")
+    let userInput = document.querySelector("#userInput")
+
+    try {
+        let data =  await axios.get(`https://ghibliapi.herokuapp.com/films`)
+        let films = data.data 
+        // let filmID = data.data.id
+
+        movieInfoDiv.innerHTML = ""
+        let title = document.createElement("h3")
+        let releaseYear = document.createElement("p")
+        releaseYear.id = "releaseYear"
+        let description = document.createElement("p")
+        description.id = "description"
+
+        const populateSelect = () => {
```

Comment:
This huge function could be broken down into three:
1. populate select 
2. event listener for the change of the select
3. event listener for submit of the form

It would make the code more readable